src/appshell/qml/NotationPage/NotationPage.qml

```diff
@@ -436,6 +447,12 @@ DockPage {
     statusBar: DockStatusBar {
         objectName: pageModel.statusBarName()
 
-        NotationStatusBar {}
+        title: qsTrc("appshell", "Notation statusbar")
```

Comment:
We need to think carefully about this wording if we want to expose it to users

@DmitryArefiev FYI